wgpu-core/src/device/mod.rs

```diff
@@ -2671,17 +2671,20 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
         use crate::backend;
         let mut callbacks = Vec::new();
 
-        #[cfg(any(
-            not(any(target_os = "ios", target_os = "macos")),
-            feature = "gfx-backend-vulkan"
-        ))]
-        self.poll_devices::<backend::Vulkan>(force_wait, &mut callbacks);
-        #[cfg(windows)]
-        self.poll_devices::<backend::Dx11>(force_wait, &mut callbacks);
-        #[cfg(windows)]
-        self.poll_devices::<backend::Dx12>(force_wait, &mut callbacks);
-        #[cfg(any(target_os = "ios", target_os = "macos"))]
-        self.poll_devices::<backend::Metal>(force_wait, &mut callbacks);
+        backends! {
```

Comment:
One thing I forgot when I posted this PR is that this is the only place where the order of statements changed with the macros. I assume this does not make a difference, but can you verify? :)

Reply:
correct, it's fine to reorder